bodhi/server/services/graphql.py

```diff
@@ -32,3 +42,23 @@ def graphql_get(request):
         str: A string "Hello World".
     """
     return "Hello World"
+
+
+@view_config()
+def graphql_view(request):
+    context = {'session': request.session}
+    return serve_graphql_request(request, schema, context_value=context)
+
+    # Setting graphiql=True
+    return serve_graphql_request(request, schema, graphiql=True, context_value=context)
```

Comment:
I also think the flag here should be `graphiql_enabled=True`.

It doesn't matter for now as the default is `True` but it will matter once we set this based on a variable :)